packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -216,3 +229,38 @@ describe('SendConfirmation', () => {
     expect(tree.queryByTestId('accountEditButton')).toBeNull()
   })
 })
+
+describe('SendConfirmation with Komenci enabled', () => {
+  const komenciEnabled = features.KOMENCI
+
+  beforeAll(() => {
+    features.KOMENCI = true
+    jest.useRealTimers()
+  })
+
+  afterAll(() => {
+    features.KOMENCI = komenciEnabled
+  })
+
+  beforeEach(() => {
+    mockedGetSendFee.mockClear()
+  })
+
+  it('renders correct modal for invitations', async () => {
+    mockedGetSendFee.mockImplementation(async () => TEST_FEE)
+
+    const store = createMockStore({
+      stableToken: {
+        balance: '200',
+      },
+    })
+
+    const tree = render(
+      <Provider store={store}>
+        <SendConfirmation {...mockInviteScreenProps} />
+      </Provider>
+    )
+
+    expect(tree).toMatchSnapshot()
```

Comment:
I'm not a big fan of only having a snapshot test, as we usually ignore big snapshots as they are hard to read.
Would much prefer to see assertions on the presence of text/buttons.

I know we're short on time here and no need to update this now though 😉

Reply:
I agree, I added some assertions for the modal, maybe we could have some for the fee as well